App/src/pages/dossier/DossierReview.tsx

```diff
@@ -649,10 +649,11 @@ export default function DossierReview() {
                                     <Heading size={"md"} color="brandBlue">
                                         Course Creation Requests:
                                     </Heading>
-                                    {/* TODO: Change the links below to the Course Details Page */}
                                     {dossierDetails?.courseCreationRequests?.map((courseCreationRequest) => (
                                         <Text key={courseCreationRequest.id} as="u">
-                                            <Link to={BaseRoutes.ManageableGroup}>
+                                            <Link
+                                                to={`/CourseDetails?subject=${courseCreationRequest.newCourse?.subject}&catalog=${courseCreationRequest.newCourse?.catalog}`}
```

Comment:
This is almost correct. Because of our versioning system, the course data returned by this endpoint won't be correct. Instead, you'll instead have to use the endpoint `/Course/id` where id is the id of the course object (meaning in this case `courseCreationRequest.newCourse?.id`). 

In the course details page, you'll have to make some check to see if it's subject/catalog that's provided, or if it's id that's provided, and query the appropriate endpoint as needed.

The same comment applies to the modification and deletion requests.

Reply:
Ok, I tried.  It'll be for the next sprint in this case.  Thanks.